ApiClient/Tests/ParsingTests.swift

```diff
@@ -1,6 +1,6 @@
 //
-//  ApiClientTests.swift
-//  ApiClientTests
+//  ParsingTests.swift
+//  ParsingTests
```

Comment:
How many times do we have to repeat `ParsingTests` until a Swift developer understands it? ;-)